packages/core/src/amazonqFeatureDev/errors.ts

```diff
@@ -98,6 +98,13 @@ export class UploadCodeError extends ToolkitError {
     }
 }
 
+export class UploadURLExpired extends ToolkitError {
+    static errorName = 'UploadURLExpired'
```

Comment:
Does this make more sense to pass in to the `name` field, adjacent to the `code`?

Reply:
It is following existing pattern.

Reply:
Can you link to examples of existing uses?

I think those may be incorrect if they are going this route. ToolkitError.name is reusing the Error.name property